gui_qt/workbench/context_library_page.py

```
@@ -84,13 +84,16 @@ def __init__(self, parent: QWidget | None = None) -> None:
         self.status_layout.root.addWidget(self.source_index_status_row)
 
-        # Phase 1 (#256): readonly status only; generation/publish land in later issues.
-        self.project_analysis_readonly_label = QLabel("只读")
+        # Phase 2 (#254): status from core module; publish/generate remain CLI-first.
+        self.project_analysis_readonly_label = QLabel("CLI")
         self.project_analysis_readonly_label.setObjectName(
             "context_project_analysis_readonly_label"
         )
         self.project_analysis_readonly_label.setToolTip(
-            "项目分析当前仅提供状态查看；生成与发布在后续阶段实现。"
+            "项目分析：状态在此只读展示。"
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)